Tests/Packet++Test/Tests/TcpTests.cpp

```diff
@@ -285,4 +286,54 @@ PTF_TEST_CASE(TcpPacketCreation2)
 	pcpp::TcpOption tcpSnackOption = tcpLayer.addTcpOption(pcpp::TcpOptionBuilder(pcpp::TCPOPT_SNACK, NULL, PCPP_TCPOLEN_SNACK));
 	PTF_ASSERT_TRUE(tcpSnackOption.isNotNull());
 	PTF_ASSERT_TRUE(tcpSnackOption.setValue(htobe32(1000)));
-} // TcpPacketCreation2
\ No newline at end of file
+} // TcpPacketCreation2
+
+
+PTF_TEST_CASE(TcpHash5Tuple)
```

Comment:
This is a very good test! I just realized there are no tests for `hash5Tuple`. Maybe we should consider creating a new `PacketUtilsTests.cpp` file and move this test there. We can also consider a similar test with UDP packets. What do you think?